packages/graphback-runtime-mongodb/src/MongoDBDataProvider.ts

```diff
@@ -170,23 +171,25 @@ export class MongoDBDataProvider<Type = any, GraphbackContext = any> implements
   }
 
   private usePage(query: Cursor<any>, page?: GraphbackPage, defaultLimit: number = 10, defaultOffset: number = 0) {
-    if (page) {
-      if (!(page.hasOwnProperty("offset")) || (page.offset === undefined)) {
-        // If no offset is supplied
-        page.offset = defaultOffset
-      } else {
-        if (page.offset < 0) {
-          throw new Error("Please use an offset of greater than or equal to 0 in queries")
-        }
-      }
-      query = query.skip(page.offset)
+    if (!page) {
+      return query.toArray();
```

Comment:
Should this just return the `query` object so people can perform pagination before they are ready to convert to a list.